### PR TITLE
Created issue template chooser 

### LINKED ISSUE #471: configure template chooser for Publiclab.Editor
### Please describe the problem (or idea)

I was trying to create a first-time-only issue, but wasn't aware of how to create it using the available first-time-only issue template.

It's more straightforward to do that in plots2.  I'm thinking we could configure the same kind of Template Chooser we have in plot2 here,
issue-temp
 so that one can easily see option to create a first-time-only issue when they click on `new issue`

> What happened just before the problem occurred? Or what problem could this idea solve?

> What did you expect to see that you didn't?

### Please show us where to look

https://publiclab.org/ ...

### What's your PublicLab.org username?

> This can help us diagnose the issue: 

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by Shulammite-Aso**
fixes #471 

Created template chooser for creating issues here in Publiclab.Editor, using the same format we have in [plots2](https://github.com/publiclab/plots2/issues )

This will make creating `first-time-only`  `bug-report ` and  `feature-request` issues easier here as well. 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt jasmine`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] PR body includes `fixes #0000`-style reference to original issue #
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by Shulammite-Aso** [comment]
I've not setup this kind of stuff before, but I simply followed the file structure in plots2 to come up with this. My thought is that it should work.

**Comment by Shulammite-Aso** [comment]
@publiclab/reviewers have a look please.

File: .github/ISSUE_TEMPLATE/--feature-request.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+---
+
+<!-- Please fill out as much of the template as you can -->
+
+<!-- Start below this comment. -->
+
+
+**Please describe the desired behavior.**
+Describe the solution you'd like.
```
Comment (VladimirMikulic) [inline .github/ISSUE_TEMPLATE/--feature-request.md:16]: I think the "feature" would be more appropriate here than "solution".
The word "solution" sounds like we are some business agency.

**Comment by Shulammite-Aso** [comment]
> Please see my suggestion. Thank you.

Right 👍

Done!

**Comment by Shulammite-Aso** [comment]
@jywarren please, do check this out too.

**Comment by jywarren** [comment]
Oh this is fantastic. Thanks so much! I'll merge today. 🎉 Great work @Shulammite-Aso!

**Pull request merged**